shanghai.example.yaml

```diff
@@ -4,6 +4,7 @@ networks:
     # serves as base for each network
     # networks can overwrite any setting here.
     nick: Shanghai
+    user: Shanghai
```

Comment:
Afaik, user and ident are separate concepts. The "user" setting would be used for authentication (such as "Server PASS", or "SASL") while the ident doesn't have any meaning.

Hexchat requires "username" to be set in its config.